app/chat_endpoint_handlers.py

```
@@ -428,7 +429,7 @@ async def chat_copy_success(chat_id: str, user_id: str, copy: ChatCopy):
         except mariadb.Error as error:
             pass
     cursor3 = sql_connection.cursor()
-    cursor3.execute(f"SELECT content, creator FROM `{mariadb_name(user_id, chat_id)}`.messages ORDER BY aid ASC LIMIT {copy.num_messages * 2};")
+    cursor3.execute(f"SELECT content, creator FROM `{mariadb_name(user_id, chat_id)}`.messages WHERE aid <= {copy.max_autoincrement} ORDER BY aid;")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)